ophyd/sim.py

```diff
@@ -1002,6 +1073,14 @@ def hw():
     jittery_motor2 = SynAxis(name='jittery_motor2',
                              readback_func=lambda x: x + np.random.rand(),
                              labels={'motors'})
+    for axis in [motor, motor1, motor2, motor3, jittery_motor1, jittery_motor2]:
+        axis.est_time = EpicsMotorEstTime(axis)
```

Comment:
I'm not sure it should be allowed to set this on an initialized object. It would be nice to have a hard guarantee that this aren't getting hot-swapped out at random times. Can `EpicsMotorEstTime` be passed in at `__init__` time?

Reply:
This is only done for the simulated devices because of the way the sim devices are defined. It is not used in normal ophyd objects.

Reply:
I agree that having the sim devices significantly different does not make them effective 'test' systems or examples.

Unfortunately when I looked at this it appears to me that the sim devices are already 'very very' different and to make them consistent would require a complete rewrite of the sim devices. I think this is outside the scope of the current PR, but perhaps could be looked at in the future.